bin/eslint-formatters-calypso/README.md

```diff
@@ -0,0 +1,82 @@
+# eslint-formatters-calypso
+
+A set of formatters to be used by ESLint and utilities to build new formatters upon.
+
+Formatters available:
+
+* `lines-modified`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch.
+* `lines-modified-and-specific-rules`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch and the rules in the `.eslines.json` file.
+* `parsing-errors`: takes the ESLint original report and outputs a new one containing only `parsing errors`.
+* `eslines`: a wrapper formatter. If the branch active is `master`, would return the results of `parsing-errors` formatter. Otherwise, would return the results of `lines-modified-and-specific-rules` formatter.
```

Comment:
In moving `eslines` to its [own repo](https://github.com/nssw/eslines/) I took the opportunity to make it [configurable](https://www.npmjs.com/package/eslines#config-file) for different scenarios and also calypso-independent. It has no concept of `master` but rather it uses an object containing particular processors to run in specific branches. For calypso, `.eslines.json` is configured to run `parsing-errors` on the branch called `master`, but you could change that to any other branch name, such as `add/eslint-formatters-calypso-testcase`, for example.

Regarding `rulesNotToDowngrade` with the `parsing-errors` processor. In terms of Calypso, I think it is OK as it is. But, it terms of `eslines`, I think this is an area worth improving and with good potential to grow. I can imagine other people building its own processors, for example. Also `rules-not-to-downgrade` could become another processor like `parsing-errors` and `lines-modified` - at that point, introducing the ability to compose processors would be the right thing to do, so, for example `.eslines.json` could look like:

    "processors": {
        "default": ["lines-modified", "rules-not-to-downgrade"],
        "master": ["parsing-errors", "rules-not-to-downgrade"],
        "my/topic-branch": "parsing-errors"
    }